lib/rspec/core/configuration.rb

```diff
@@ -471,6 +473,9 @@ def initialize
       # Used to set higher priority option values from the command line.
       def force(hash)
         ordering_manager.force(hash)
+        [:deprecation_stream, :output_stream].each do |stream|
+          hash[stream] = prepare_stream(hash[stream]) if hash.key?(stream)
```

Comment:
This is mutating the passed in `hash`.  In general, it's a bad idea to mutate arguments passed to methods as it leads to unpleasant surprises.